transport/protocol/src/msg/processor.rs

```diff
@@ -525,6 +525,7 @@ impl PacketInteraction {
                 }
             });
 
+        // the task is terminated by closing all TX instances
```

Comment:
We should add these comments to all the above `spawn` calls, where the JH is not stored.